am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/InvalidArgumentsProvider.java

```diff
@@ -37,8 +41,9 @@
         List<Arguments> combinations = new ArrayList<>();
 
         Type[] parameterTypes = context.getRequiredTestMethod().getGenericParameterTypes();
+        Annotation[][] annotations = context.getRequiredTestMethod().getParameterAnnotations();
```

Comment:
@ValidAttributeSecurityClassification **@exampleAnotherAnnotation**
        Map<JsonPointer, SecurityClassification> attributeSecurityClassifications if we have multiple Annotations for field will  this code works?

Reply:
👍 , can handle multiple annotations as discussed